libs/easyloggingpp/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=easyloggingpp
+PKG_VERSION:=9.97.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
This is wrong. :-( The tarball is going to be downloaded only as version in the dl folder., so no name and thats not wanted.